src/batou/tests/test_config.py

```diff
@@ -129,7 +129,7 @@ def test_config_exceptions_orderable(env):
         batou.SuperfluousComponentSection.from_context("asdf"),
         batou.SuperfluousSecretsSection.from_context("asdf"),
         batou.CycleErrorDetected.from_context(ValueError()),
-        batou.NonConvergingWorkingSet.from_context([c]),
+        batou.NonConvergingWorkingSet.from_context([c.root]),
```

Comment:
Interestingly, this uncovers a bug in the test suite (NonConveringWorkingSet is constructed with ComponentRoots, not Components